test/CMakeLists.txt

```diff
@@ -37,7 +41,11 @@ if(TARGET lldb-mi)
 endif()
 
 if(NOT LLDB_BUILT_STANDALONE)
-  list(APPEND LLDB_TEST_DEPS yaml2obj dsymutil)
+  if (CMAKE_SYSTEM_NAME MATCHES "Darwin")
+    list(APPEND LLDB_TEST_DEPS yaml2obj dsymutil)
+  else()
```

Comment:
To the best of my understanding, `yaml2obj` is available anywhere. Any reason why you're limiting this change to Darwin? (Also, yaml2obj is required to run the `lit` tests).

Reply:
If you look closer, this pull request does not change anything about `yaml2obj` or its usage, just `dsymutil`.